pkg/k8s/crd/config/edgedevice/bases/shifu.edgenesis.io_telemetryservices.yaml

```diff
@@ -42,7 +42,7 @@ spec:
                   type: string
                 type: object
               serviceSettings:
-                description: ProtocolSettings defines protocol settings when connecting
+                description: ServiceSettings defines protocol settings when connecting
```

Comment:
do we need to change the description?  "ServiceSettings defines protocol settings when connecting to an Edge Device"

""ServiceSettings define the service settings when connecting to an Edge Device"

Reply:
yep,actrually the describe is a mistake